server/internal/server/completions.go

```diff
@@ -72,7 +73,12 @@ func (s *S) CreateChatCompletion(
 		return
 	}
 
+	taskID, err := id.GenerateID("inf_", 22)
```

Comment:
just checking, most IDs are 24 in length, but not aligned here?

Reply:
good call! Let me fix.

Reply:
```suggestion
	taskID, err := id.GenerateID("inf_", 24)
```